### PR TITLE
Add errors group to input output tab

### PR DESCRIPTION
**Pull request by azeghers**
Related to camunda/camunda-modeler#2070

This PR adds the UI component of the `camunda:ErrorEventDefinition` elements in the properties panel, in the input/output tab.

2 missing features to note :
-Copying and pasting a service task with a camunda:ErrorEventDefinition that references a bpmn:Error doesn't apply the reference on paste.
-Changing the type of a service task with camunda:ErrorEventDefninitions won't delete them, just hide the errors group in the input/output tab

Those features will be added at a later stage.

__Acceptance Criteria__

* [ ] Corresponds to the concept 
* [ ] Corresponds to the design 

__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

**Comment by pinussilvestrus** [comment]
Maybe we should squash the commits beforehand.

File: lib/provider/bpmn/parts/implementation/ErrorEventDefinition.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ module.exports = function(group, element, bpmnFactory, errorEventDefinition, tra
   group.entries = group.entries.concat(
     elementReferenceProperty(element, errorEventDefinition, bpmnFactory, translate, {
       id: 'error-element-name',
-      label: translate('Global Error Name'),
+      label: translate('Name'),
```
Comment (pinussilvestrus) [inline lib/provider/bpmn/parts/implementation/ErrorEventDefinition.js:22]: the labels differ from the sketch in the referenced issue (https://github.com/bpmn-io/bpmn-js-properties-panel/issues/422)

image

but I guess the sketch got outdated in the meanwhile.

File: lib/provider/camunda/CamundaPropertiesProvider.js [hunk 2]
```diff
@@ -420,9 +421,22 @@ function createInputOutputTabGroups(element, bpmnFactory, elementTemplates, tran
 
   outputParameters(outputParametersGroup, element, bpmnFactory, elementTemplates, translate);
 
+  var errorsGroup = {
+    id: 'errors',
+    label: translate('Errors'),
+    entries: [],
+
+    enabled: function(element, node) {
+      return is(element, 'bpmn:ServiceTask') && getBusinessObject(element).type === 'external';
```
Comment (pinussilvestrus) [inline lib/provider/camunda/CamundaPropertiesProvider.js:430]: I wonder if we would get safer if we use [internal helpers](https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/lib/helper/ImplementationTypeHelper.js#L137) for this.

```js
return ImplementationTypeHelper.getImplementationType(element) === 'external';
```

File: styles/properties.less [hunk 3]
```diff
@@ -520,18 +524,51 @@ label.bpp-hidden {
   opacity: 0.7;
 }
 
+.bpp-collapsible-error .bpp-collapsible__description:not(:empty) {
```
Comment (pinussilvestrus) [inline styles/properties.less:527]: Maybe we can standardize to not have selectors for any type of collapsible item. But that's for sure not part of this PR.

File: test/spec/provider/camunda/ErrorsPropsSpec.js [hunk 4]
```diff
@@ -0,0 +1,578 @@
...
+        var collapsibleDescription = getCollapsibleDescription(collapsible);
+
+        // then
+        expect(collapsibleDescription.textContent).to.eql('${expression}');
+      }));
+
+    });
+
+
+    describe('create camunda:ErrorEventDefinition', function() {
```
Comment (pinussilvestrus) [inline test/spec/provider/camunda/ErrorsPropsSpec.js:177]: These (and other as for the errors) tests are more testing the DOM changes (is collapsible existent, has select box this amount of entries). I think we should also check whether the updates were made on the business object level.

```js
describe('create camunda:ErrorEventDefinition', function() {

  describe('in the DOM', function () {
    
    it('should do');


    it('should undo');


    it('should redo');

  });


  describe('on the businessObject', function () {
    
    it('should do');


    it('should undo');


    it('should redo');

  });

});
```

**Review by pinussilvestrus — COMMENTED** [review]
Some things we should consider in the implementation & tests, besides, good job so far 👏

File: lib/provider/bpmn/parts/implementation/ErrorEventDefinition.js (hunk 1 above)
Comment (azeghers) [reply lib/provider/bpmn/parts/implementation/ErrorEventDefinition.js:22]: The sketch is outdated yeah, we settled for naming without the "Error-" prefix.

File: lib/provider/camunda/CamundaPropertiesProvider.js (hunk 2 above)
Comment (azeghers) [reply lib/provider/camunda/CamundaPropertiesProvider.js:430]: Sure thing!

File: styles/properties.less (hunk 3 above)
Comment (azeghers) [reply styles/properties.less:527]: Yeah I wasn't too sure about that, would you see an immediate change to apply?

File: test/spec/provider/camunda/ErrorsPropsSpec.js (hunk 4 above)
Comment (azeghers) [reply test/spec/provider/camunda/ErrorsPropsSpec.js:177]: Will add businessObject-related tests.

File: styles/properties.less (hunk 3 above)
Comment (pinussilvestrus) [reply styles/properties.less:527]: not immediately no, let's keep it in our mind

also cc @barmac

**Comment by andreasgeier** [comment]
Checked the latest artifact right now - looks and behaves as expected. ✅

File: styles/properties.less (hunk 3 above)
Comment (barmac) [reply styles/properties.less:527]: Not sure what we can do about it right now. I hope we move someday out of the current collapsible trick.

**Review by pinussilvestrus — APPROVED** [review]
Changes LGTM 👍

File: test/spec/provider/camunda/ErrorsPropsSpec.js (hunk 4 above)
Comment (pinussilvestrus) [reply test/spec/provider/camunda/ErrorsPropsSpec.js:180]: cool!
